Unable to build CLI locally

# Bug Report

**Current behavior**

On MacOS 10.13.3
On `yarn build` inside directory `cli/packages/prisma-cli-engine`:

```
▶ yarn build
yarn run v1.3.2
$ tsc -d
src/Client/Client.test.ts(3,29): error TS2307: Cannot find module 'prisma-yml'.
src/Client/Client.ts(19,53): error TS2307: Cannot find module 'prisma-yml'.
src/Command.ts(10,52): error TS2307: Cannot find module 'prisma-yml'.
src/StatusChecker.ts(7,29): error TS2307: Cannot find module 'prisma-yml'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
**Reproduction**

![delete](https://user-images.githubusercontent.com/17485128/36347807-34bc37c0-141e-11e8-8be6-61611bdea966.jpeg)


**Expected behavior?**

Successful build

--- a/cli/CONTRIBUTING.md
+++ b/cli/CONTRIBUTING.md
@@ -4,6 +4,8 @@
 $ git clone git@github.com:graphcool/prisma.git
 $ cd prisma/cli
 $ yarn install
+$ cd packages/prisma-yml
+$ yarn install
 $ cd packages/prisma-cli-engine
 $ yarn build
 $ cd ../prisma-cli-core

